src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        Colors randomColors = Colors.values()[index];
+        return randomColors.toString();
```

Comment:
Use `randomColors.name()` instead of `randomColors.toString()` to get the string representation of the enum constant. This is a more reliable method as per the checklist recommendation.